packages/experiments/src/components/Chiclet/ChicletCard.base.tsx

```diff
@@ -51,12 +49,12 @@ export class ChicletCardBase extends BaseComponent<IChicletCardProps, {}> {
         tabIndex={tabIndex}
         role={role}
         onClick={actionable ? this._onClick : undefined}
-        className={css('ms-ChicletCard', className, mergeStyles(this._classNames.root))}
+        className={this._classNames.root}
```

Comment:
@natalieethell I think this may have been your code. We should not call `mergeStyles` on classnames that were already merged. It is pretty much wasteful. We should also not need to use the `css` helper when a `*.style.ts` file exists. All logic for what goes in the classnames can live there.